### PR TITLE
Changed to new command for installing from tsd.json

### PR DESCRIPTION
**Pull request by perkastman**
Small documentation update, change from tsd install to tsd reinstall for installation from tsd.json file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ $ tsd query jquery.*
 $ tsd query "jquery.*"
 
 #install all definitions from tsd.json
-$ tsd install
+$ tsd reinstall
```
Comment (Diullei) [inline README.md]: Hi @perkastman! The `install` command performs like the `reinstall` command in this case. You can update the doc informing that the two ways are correct.

Comment (perkastman) [inline README.md]: Hmm, so why do I get this error message when I run tsd install?
pass at least one query pattern
VError: pass at least one query pattern

With reinstall it works as excepted.

Comment (Diullei) [inline README.md]: What `tsd` version are you using? Could you run `tsd --version` please?

Comment (perkastman) [inline README.md]: Sorry, should have mentioned that. I'm running tsd 0.6.0

Comment (perkastman) [inline README.md]: Updated to 0.6.4 and there it works, so never mind I guess :-)

Comment (Diullei) [inline README.md]: Nice! Here (https://github.com/DefinitelyTyped/tsd/releases) you can see the latest published versions :)

**Pull request closed without merging**